CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ cmake_minimum_required(VERSION 3.2.2)
 # Project Setup - modify to match project naming
 ## Source code for a simple command-line executable for a dynamic library will be generated from the project name.
 ## The command-line and library names will be based off the project name.
-project(cpp-hocon VERSION 0.1.0)
+project(cpp-hocon VERSION 0.1.2)
```

Comment:
This got missed when we tagged. When is the correct time to fix it, if not part of this PR?